From bc072f6f913afe9a250bb106b3bc92c48f30dcd1 Mon Sep 17 00:00:00 2001
From: Wu Liang feng <wulf@rock-chips.com>
Date: Wed, 6 Jul 2016 10:35:47 +0800
Subject: [PATCH] ARM64: configs: rockchip_cros_defconfig: enable extcon
 support

This allows monitoring external connectors by userspace
via sysfs and uevent and supports external connectors with
multiple states. And extcon notifier mechanism is also
useful.

Change-Id: I670da8efbe20f8f12529e9e062fc38b514524dc3
Signed-off-by: Wu Liang feng <wulf@rock-chips.com>
---
 arch/arm64/configs/rockchip_cros_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_cros_defconfig b/arch/arm64/configs/rockchip_cros_defconfig
index 584a12d04e09..d85f34bff7c9 100644
--- a/arch/arm64/configs/rockchip_cros_defconfig
+++ b/arch/arm64/configs/rockchip_cros_defconfig
@@ -544,6 +544,7 @@ CONFIG_DEVFREQ_GOV_SIMPLE_ONDEMAND=y
 CONFIG_DEVFREQ_GOV_PERFORMANCE=y
 CONFIG_DEVFREQ_GOV_POWERSAVE=y
 CONFIG_DEVFREQ_GOV_USERSPACE=y
+CONFIG_EXTCON=y
 CONFIG_IIO=y
 CONFIG_ROCKCHIP_SARADC=y
 CONFIG_PWM=y
-- 
2.35.3

